pymc3/gp/gp.py

```diff
@@ -793,7 +794,7 @@ def conditional(self, name, Xnew, pred_noise=False, given=None, **kwargs):
         return pm.MvNormal(name, mu=mu, chol=chol, shape=shape, **kwargs)
 
 
-@conditioned_vars(["Xs", "y", "noise"])
+@conditioned_vars(["Xs", "y", "sigma"])
 class MarginalKron:
```

Comment:
Any reason you didn't subclass the base class (even though it doesn't do much)?

Reply:
Only because at one point I had not implemented a `cov_func`, but instead only had a list of `cov_funcs`. Now that I've made one by multiplying the separate ones together, I guess there wouldn't be any harm in subclassing.

Reply:
Sounds good to me